platforms/cpuinfo.go

```diff
@@ -37,108 +32,12 @@ var cpuVariantOnce sync.Once
 func cpuVariant() string {
 	cpuVariantOnce.Do(func() {
 		if isArmArch(runtime.GOARCH) {
-			cpuVariantValue = getCPUVariant()
+			var err error
+			cpuVariantValue, err = getCPUVariant()
+			if err != nil {
+				log.L.Errorf("Error getCPUVariant for OS %s : %v", runtime.GOOS, err)
```

Comment:
style: we generally don't put spaces before `:`.

Reply:
Thanks for pointing out and maintain the high standard! 

I have updated accordingly with the new commit.